src/fil_cns/validation.rs

```diff
@@ -56,7 +56,8 @@ pub(in crate::fil_cns) async fn validate_block<DB: Blockstore + Sync + Send + 's
 
     let base_tipset = chain_store
         .tipset_from_keys(header.parents())
-        .map_err(to_errs)?;
+        .map_err(to_errs)?
+        .ok_or_else(|| to_errs(crate::chain::store::Error::NotFound(Default::default())))?;
```

Comment:
`crate::chain::store::Error::NotFound` is a bit hard on the eyes. What about importing `crate::chain::Error` first?

Reply:
Fixed.